rpcs3/rpcs3qt/game_list_frame.cpp

```diff
@@ -40,7 +46,7 @@ game_list_frame::game_list_frame(std::shared_ptr<gui_settings> guiSettings, std:
 	m_Icon_Color      = m_gui_settings->GetValue(gui::gl_iconColor).value<QColor>();
 	m_colSortOrder    = m_gui_settings->GetValue(gui::gl_sortAsc).toBool() ? Qt::AscendingOrder : Qt::DescendingOrder;
 	m_sortColumn      = m_gui_settings->GetValue(gui::gl_sortCol).toInt();
-	m_hidden_list     = m_gui_settings->GetValue(gui::gl_hidden_list).toStringList().toSet();
+	m_hidden_list     = list_to_set(m_gui_settings->GetValue(gui::gl_hidden_list).toStringList());
```

Comment:
Use the following, unless it's deprecated as well
```suggestion
	m_hidden_list     = QSet<QString>::fromList(m_gui_settings->GetValue(gui::gl_hidden_list).toStringList());
```